nuxeo-retention-web/pom.xml

```diff
@@ -0,0 +1,44 @@
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>org.nuxeo.retention</groupId>
+    <artifactId>nuxeo-retention-parent</artifactId>
+    <version>11.0.0-SNAPSHOT</version>
```

Comment:
should be ` <version>11.4-SNAPSHOT</version>` otherwise it will not compile, tested on local